source/common/runtime/runtime_impl.cc

```diff
@@ -239,6 +255,17 @@ SnapshotImpl::Entry SnapshotImpl::createEntry(const std::string& value) {
   return entry;
 }
 
+bool SnapshotImpl::parseEntryBooleanValue(Entry& entry) {
+  if (absl::EqualsIgnoreCase(entry.raw_string_value_, "true")) {
```

Comment:
Is it worth L and R string trimming here and below? I think it would be more user friendly if there is a CR/LF in the file? WDYT?